tests/acceptance/project-conversations-test.js

```diff
@@ -4,6 +4,8 @@ import moduleForAcceptance from 'code-corps-ember/tests/helpers/module-for-accep
 import { authenticateSession } from 'code-corps-ember/tests/helpers/ember-simple-auth';
 import page from 'code-corps-ember/tests/pages/project/conversations';
 
+const hourOfMiliseconds = 3600 * 1000;
```

Comment:
Can you just write this as hour? Means effectively the same thing and is easier to read quickly.